src/GOODFFunctions.h

```diff
@@ -26,6 +26,12 @@
 #include <wx/filename.h>
 #include <vector>
 #include "GOODF.h"
+// macro to both show and raise the LogWindow
+#define SHOWLOGWINDOW do { \
+	wxLogWindow* w = ::wxGetApp().m_frame->GetLogWindow(); \
+	w->Show(true); \
+	w->GetFrame()->Raise(); \
```

Comment:
In remote testing of a .organ file with all pipes missing, `w->GetFrame()->Raise();` is quite expensive and GoOdf appears hung if there are a lot of SHOW_LOG_WINDOW calls.  Since the LogWindow is now raised in 1f1178dd639f28a31996fc781bfd97f3cc3437da